corehq/apps/domain/management/commands/generate_malt_table.py

```diff
@@ -0,0 +1,84 @@
+from django.core.management.base import BaseCommand
+
+from corehq.apps.app_manager.models import Application
+from corehq.apps.domain.models import Domain
+from corehq.apps.smsforms.app import COMMCONNECT_DEVICE_ID
+from corehq.apps.sofabed.models import FormData
+
+import csv
+import datetime
+
+
+class Command(BaseCommand):
+    help = 'Generate malt table for June 2015'
+
+    def write_data(self, csv_writer, start_date, end_date):
+        for domain in Domain.get_all():
+            for user in domain.all_users():
+                forms = FormData.objects.exclude(device_id=COMMCONNECT_DEVICE_ID).filter(
+                    user_id=user._id,
+                    domain=domain.name,
+                    received_on__range=(start_date, end_date)
+                )
+                num_of_forms = forms.count()
+                is_web_user = user.doc_type == 'WebUser'
+                apps_submitted_for = forms.values_list('app_id').distinct()
+                apps_submitted_for = [app_id for (app_id,) in apps_submitted_for]
+
+                for app_id in apps_submitted_for:
+                    wam, pam = self._wam_pams(app_id)
+                    row = [
+                        user._id,
+                        user.username,
+                        user.email,
+                        app_id,
+                        is_web_user,
+                        str(start_date),
+                        domain.name,
+                        num_of_forms,
+                        wam,
+                        pam
+                    ]
+                    csv_writer.writerow(row)
```

Comment:
I might propose that we write this to the database so we can query and update it over time more easily